### ISSUE TITLE
Couldn't remove user from role even with MANAGED_POLICIES

### ISSUE THREAD

**Issue by laulpogan** [datahub-project/datahub#7012]
https://datahubspace.slack.com/archives/CV2UXSE9L/p1673379681493409?thread_ts=1673304287.257529&cid=CV2UXSE9L

Hi everyone. I try to use the permissions settings to configure permissions control via ui, but once add a user in a Role i couldn't remove him.
In the Users & Groups tab i had the same issue.

A way that i found to finish my task was remove a user that was editor and i change him to admin for example, and add him to group that i create with all privileges, after a new login when he appears as a user again.

Summing up, once add a role to a user by ui (in Settings >Manage Permissions > Permissions >  ADD USERS button in Roles tab, or Settings > Manage Users & Groups > Users tab dropdown) and change this role, the old role persist.
Other issue is that add roles for a user by Roles tab, the same user can have more than one role. (edited) 
white_check_mark
eyes
thanks_bear

**Comment by aditya-radhakrishnan** [comment]
This issue is because the Neo4j Implementation of the GraphService does not remove relationship edges. This needs to be implemented and will fix this issue. This IS working for Elastic!

**Comment by jjoyce0510** [comment]
This is pending tech debt. Neo4j support is compromised by this change
